container-toolkit/supported-platforms.md

```diff
@@ -16,7 +16,12 @@ backlinks: none
 
 ## Linux Distributions
 
-Supported Linux distributions are listed below:
+The NVIDIA Container Toolkit has been qualified against the following platforms.
+
+Note that the installation packages are generally forward-compatible and the
+omission of newer distributions from this list below does not indicate that the
+NVIDIA Container Toolkit cannot be used there. Specific issues with newer
+distributions should be reported.
```

Comment:
I think this can be rephrased more concisely.

```
NVIDIA Container Toolkit is generally forward compatible and works successfully on newer versions of the operating systems in the following table.
```

I've puzzled through the thought of directing readers to report issues from this section of the text, and I can't think why it's more compelling to urge that action here than in any other section.   If you prefer to keep it, consider something like "Open an [issue](https://github.com/NVIDIA/nvidia-container-toolkit/issues) in the toolkit repository to report problems."